common/v2/features/Settings/Settings.tsx

```diff
@@ -11,15 +11,18 @@ import { AddressBookPanel, AddToAddressBook, GeneralSettings, DangerZone } from
 import settingsIcon from 'common/assets/images/icn-settings.svg';
 import { IS_ACTIVE_FEATURE } from 'v2/config';
 
-const SettingsHeading = styled(Heading)`
+const SettingsHeading = styled(Heading)<{ forwardedAs?: string }>`
   display: flex;
   align-items: center;
-  margin-bottom: 22px;
-  color: #163150;
+  margin-bottom: 24px;
+  font-weight: bold;
+  margin-top: 0;
 `;
 
 const SettingsHeadingIcon = styled.img`
-  margin-right: 12px;
+  margin-right: 24px;
```

Comment:
should these use `${SPACING}` variables?

Reply:
Well we don't have 16px increments yet and I hope this style will change when they see it, so I'm fine leaving them inline atm.